### DISCUSSION TITLE
Unable to Apply for Github Student

### DISCUSSION THREAD

**Discussion by mohammadhashimcodes** [community/community#124531] [GitHub Education]
### Hi everyone,

I am applying for Github education since last week I have created my account with my university email id and instead of that I am getting only one rejection while uploading photo that I am not in near campus while i am in my campus and not using vpn and location enabled.

**Comment by FrootLoopsBowl** [comment]
Hi,

I faced a similar issue when applying for GitHub Education. Here are a few tips that might help:

1. Double-check your photo: Make sure the photo clearly shows you on campus. If possible, include identifiable campus landmarks.
2. Verify your location settings: Ensure your device's location services are turned on and that your browser has permission to access your location.
3. Use a different browser or device: Sometimes, switching browsers or devices can help resolve location detection issues.
4. Contact GitHub Support: If the issue persists, reach out to GitHub Education support. Explain your situation and provide any additional proof of your campus presence.

Hope this helps, and good luck with your application!

Froot

**Comment by dcaayushd** [comment]
Possible Solutions
1. Ensure that my location services are enabled and working correctly.
2. Try uploading the photo from different locations within the campus.
3. Reach out to the GitHub Education support team for further assistance.

Please let me know if you need any additional information or if there are any other steps I can take to resolve this issue.
